src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -66,23 +73,137 @@ const styles = StyleSheet.create({
 });
 
 export interface AnonymousInputMessageProps {
-  onSendButtonClicked: (message: string) => void;
+  onSendButtonClicked: (message: string, attachments: any) => void;
   type: 'SIGNED' | 'ANONYMOUS';
 }
 
 const AnonymousInputMessage = ({onSendButtonClicked, type}: AnonymousInputMessageProps) => {
   const refEmoji = React.useRef(null);
+  const refAttachment = React.useRef(null);
   const [text, setText] = React.useState('');
+  const [isLoadingUploadImageMedia, setIsLoadingUploadImageMedia] = React.useState(false);
+  const [isLoadingUploadImageCamera, setIsLoadingUploadImageCamera] = React.useState(false);
+
+  const handleUploadMedia = async (paths) => {
+    setIsLoadingUploadImageMedia(true);
+
+    const resultUrls = paths.map((path) => ({
+      type: 'image',
+      asset_url: path,
+      thumb_url: path,
+      myCustomField: 'image'
+    }));
+    console.warn('resultUrls', resultUrls);
+    onSendButtonClicked(' ', resultUrls);
+    setIsLoadingUploadImageMedia(false);
+  };
+
+  const handleUploadCamera = async (path) => {
+    setIsLoadingUploadImageCamera(true);
+
+    const resultUrls = [];
+    const uploadedImageUrl = await ImageUtils.uploadImageWithoutAuth(path);
+    resultUrls.push({
+      type: 'image',
+      asset_url: uploadedImageUrl,
+      thumb_url: uploadedImageUrl,
+      myCustomField: 'image'
+    });
+
+    onSendButtonClicked(' ', resultUrls);
+    setIsLoadingUploadImageCamera(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the function `handleUploadCamera` lacks error handling. It's important to add a try-catch block here as well.



<!-- This is an auto-generated comment by OSS CodeRabbit -->